packages/shared/src/lib/core/wallet/actions/send/createEvmChainToEvmChainTransaction.ts

```diff
@@ -30,7 +30,7 @@ export function createEvmChainToEvmChainTransaction(
     ) {
         const tokenAmount = getAmountAndTokenFromSendFlowParameters(sendFlowParameters)
         const token = tokenAmount.token
-        const amount = tokenAmount.amount
+        const amount = tokenAmount.amount?.toString()
```

Comment:
if this is required to be a string somewwhere deeper, we should cast it there. i.e. if this needs to be a string as the output of buildEvmTransactionDataForToken then we should do the type tansformation in there.

Reply:
just looked and its mostly used / casted as a bigint deeper anyway

Reply:
The reason for this was that the calculate gas expects it as a string, but I convert the bigint there.